docs/basic-docs/atomicdex-api-legacy/unban_pubkey.md

```diff
@@ -0,0 +1,110 @@
+# unban\_pubkey
+
+**unban_pubkey type data**
+
+The `unban_pubkey` method will remove all currently pubkeys from your ban list, or specific pubkeys from a user defined list.
```

Comment:
It seems that "banned" is missing in "currently pubkeys".

Reply:
fixed, thanks.